pkgs/development/libraries/sfml/default.nix

```diff
@@ -15,6 +15,7 @@ stdenv.mkDerivation rec {
   buildInputs = [ cmake libX11 freetype libjpeg openal flac libvorbis glew
                   libXrandr libXrender udev xcbutilimage
                 ];
+  cmakeFlags = "-DSFML_INSTALL_PKGCONFIG_FILES=yes";
```

Comment:
It is generally a good idea to make `cmakeFlags` (and similar fields) lists. That is,
```
cmakeFlags = [ "-DSFML_INSTALL_PKGCONFIG_FILES=yes" ];
```
This helps, for example, when you want to override the package expression and add additional cmake flags.